### ISSUE TITLE
Fix the failing pytest version update

### ISSUE THREAD

**Issue by bidoubiwa** [meilisearch/docs-scraper#287]
See #275 

The tests are failing since the new version, it should be investigated and fixed.

**Comment by sanders41** [comment]
I don't think it is related to pytest specifically. The pylint check also fails to run. All the errors are related to new things in Python 3.11 and an older version of python is being run in the tests. Exception groups are new and 3.11 and it also has a built in toml parser.

Potentially it is something with the `Pipfile.lock` and python versions not playing nice together.

**Comment by bidoubiwa** [comment]
Ah it is probably related to this #285 I suppose

**Comment by sanders41** [comment]
I think [this](https://github.com/meilisearch/meilisearch-python/pull/612#issuecomment-1336299347) is the issue. If it turns out my theory is correct there I don’t think my PR will solve the problem, and I don’t currently have any other ideas.
